tools/install_tensorflow_py.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
I don't see either of these install_tensorflow scripts referred to anywhere else in your PR.
They should at least be mentioned in an error message with installation instructions, for when
the example scripts are invoked but TensorFlow was not installed.